src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -25,13 +33,14 @@ def __init__(self, endpoint):
   def get_max_payload_bytes(self):
     return None
 
-  def send(self, payload):
+  def send(self, file_path):
     try:
-      requests.post(
-        self.endpoint,
-        data=payload,
-        headers={'Content-Type': 'application/x-thrift'},
-      )
+      command = 'curl -v -X POST -H "Content-Type: application/json" --data @{} {}'.format(file_path, self.endpoint)
+      args = command.split(' ')
+      p = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
We probably want to connect all of stdin, stdout, and stderr to `/dev/null` by doing something like:
`Popen(args, stdin=os.devnull, stdout=os.devnull, stderr=os.devnull)`